client/src/components/History/Content/Dataset/DatasetActions.vue

```diff
@@ -55,7 +55,7 @@
                 <b-button
                     v-if="showHighlight"
                     class="highlight-btn px-1"
-                    title="Show Inputs for this item"
+                    title="Show Related Items"
```

Comment:
Thanks, I saw this text was out of date when looking at the other related filter PR, glad to see it fixed here!  :)